Core/Chat/TwitchChat.cs

```diff
@@ -179,7 +179,10 @@ private async void WhisperReceived(object? sender, OnWhisperReceivedArgs e)
                 fromMessage: true,
                 updatedAt: _clock.GetCurrentInstant()
             ));
-            var message = new Message(user, messageText, source);
+            // TODO whispers don't have "RawIrcMessage" yet.
+            // The required change is merged, but not released yet: https://github.com/TwitchLib/TwitchLib.Client/pull/159
+            var rawIrcMessage = (twitchLibMessage as ChatMessage)?.RawIrcMessage ?? string.Empty;
```

Comment:
does this mean that tags won't be available for this? that seems like it'd potentially cause issues

Reply:
actually this means that whispers cannot be forwarded at all... I totally forgot about that. This probably just requires a TwitchLib release

Reply:
they did have a release, I just included the wrong dependency. Fixed it